src/View/mainpage/IsodoseTab.py

```diff
@@ -166,5 +166,5 @@ def on_loaded_iso2roi(self):
         Closes the progress window and refreshes
         the main screen.
         """
-        self.request_update_ui.emit()
+        self.request_update_ui.emit((self.patient_dict_container.get('dataset_rtss'), {"draw": None}))
```

Comment:
```suggestion
        self.request_update_ui.emit((
            self.patient_dict_container.get('dataset_rtss'), {"draw": None}))
```

PEP 8 compliance - line too long.